mautic/mautic
# Fix category type for Marketing messages

Closes #8516
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | yes
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #8516
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
I noticed Marketing messages has bad category type. It was store as 0, we expect messages (as other categories type)
This PR fixed it and try prevent for that situation.
Also migrate all 0 categories type to messages.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Go to categories
2.  Create category with type messages
3.  Go to your database, check categories table and see type is 0.
4. We expect messages

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2.  Create category with type Marketing messages
3.  Go to your database, check categories table and see type is __messages__
